.github/schemas/branding-manifest.schema.json

```diff
@@ -0,0 +1,144 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Branding Data Manifest",
+  "description": "Contract interface between hoverkraft-tech/branding and hoverkraft-tech/landing-page repositories",
+  "type": "object",
+  "required": ["version", "timestamp", "commit", "colors", "logos"],
+  "properties": {
+    "version": {
+      "type": "string",
+      "description": "Semantic version of the branding data (e.g., 1.0.0)",
+      "pattern": "^\\d+\\.\\d+\\.\\d+$"
+    },
+    "timestamp": {
+      "type": "string",
+      "description": "ISO 8601 timestamp when the branding data was generated",
+      "format": "date-time"
+    },
+    "commit": {
+      "type": "string",
+      "description": "Git commit SHA of the branding repository",
+      "pattern": "^[0-9a-f]{40}$"
+    },
+    "colors": {
+      "type": "object",
+      "description": "Brand color definitions",
+      "required": ["file", "count"],
+      "properties": {
+        "file": {
+          "type": "string",
+          "description": "Path to colors.css file in the artifact",
+          "example": "colors.css"
+        },
+        "count": {
+          "type": "integer",
+          "description": "Number of color tokens defined",
+          "minimum": 0
+        }
```

Comment:
We dont want to get a css file but a list of colors with desscription, hexa code, usage...